docs/data/stashcache/run-stashcache-container.md

```diff
@@ -0,0 +1,184 @@
+Running Stashcache in a Container
+=================================
+
+The OSG operates the [StashCache data federation](../stashcache/overview/), which
+provides organizations with a method to distribute their data in a scalable manner to thousands of jobs without needing
+to pre-stage data across sites or operate their own scalable infrastructure.
+
+[Stash Caches](../install-cache/) transfer data to clients such as jobs or users. A set of caches are operated across the OSG for the benefit of nearby sites;
+in addition, each site may run its own cache in order to reduce the amount of data transferred over the WAN. This documents how to run stashcache in a Docker container.
+
+Overview
+--------
+
+The minimal decisions when running a stashcache are:
```

Comment:
```suggestion
Before starting the installation process, consider the following points:
```

May as well use the same language as we do for the install guides. It's also worth noting that the user will need to have docker running and we assume they have root access so we should include those as bullet items

Reply:
Not really we only assume root for the systemd stuff I can place it there.

Reply:
Well, technically you don't need root for that either. But to run a docker container you need to be part of the `docker` group at the very least, right?

Reply:
@brianhlin what about now? can you review it ?